rclcpp_components/include/rclcpp_components/component_manager.hpp

```diff
@@ -57,31 +62,34 @@ class ComponentManager : public rclcpp::Node
   using ComponentResource = std::pair<std::string, std::string>;
 
   ComponentManager(
-    std::weak_ptr<rclcpp::executor::Executor> executor);
+    std::weak_ptr<rclcpp::executor::Executor> executor,
+    std::string node_name = "ComponentManager");
```

Comment:
Would it not make sense to have also a constructor that forwards the namespace and node options arguments?

Reply:
I agree that would probably make sense in this case to make it general purpose.